filebeat/etc/beat.yml

```diff
@@ -1,176 +1,175 @@
+    # Note: After is the equivalent to previous and before is the equivalent to to next in Logstash
+    #multiline.match: after
+
+    # The maximum number of lines that are combined to one event.
+    # In case there are more the max_lines the additional lines are discarded.
+    # Default is 500
+    #multiline.max_lines: 500
+
+    # After the defined timeout, an multiline event is sent even if no new pattern was found to start a new event
+    # Default is 5s.
+    #multiline.timeout: 5s
+
+    # Setting tail_files to true means filebeat starts reading new files at the end
+    # instead of the beginning. If this is used in combination with log rotation
+    # this can mean that the first entries of a new file are skipped.
+    #tail_files: false
+
+    # Backoff values define how aggressively filebeat crawls new files for updates
+    # The default values can be used in most cases. Backoff defines how long it is waited
+    # to check a file again after EOF is reached. Default is 1s which means the file
+    # is checked every second if new lines were added. This leads to a near real time crawling.
+    # Every time a new line appears, backoff is reset to the initial value.
+    #backoff: 1s
+
+    # Max backoff defines what the maximum backoff time is. After having backed off multiple times
+    # from checking the files, the waiting time will never exceed max_backoff independent of the
+    # backoff factor. Having it set to 10s means in the worst case a new line can be added to a log
+    # file after having backed off multiple times, it takes a maximum of 10s to read the new line
+    #max_backoff: 10s
+
+    # The backoff factor defines how fast the algorithm backs off. The bigger the backoff factor,
+    # the faster the max_backoff value is reached. If this value is set to 1, no backoff will happen.
+    # The backoff value will be multiplied each time with the backoff_factor until max_backoff is reached
+    #backoff_factor: 2
+
+    # This option closes a file, as soon as the file name changes.
+    # This config option is recommended on windows only. Filebeat keeps the files it's reading open. This can cause
+    # issues when the file is removed, as the file will not be fully removed until also Filebeat closes
+    # the reading. Filebeat closes the file handler after ignore_older. During this time no new file with the
+    # same name can be created. Turning this feature on the other hand can lead to loss of data
+    # on rotate files. It can happen that after file rotation the beginning of the new
+    # file is skipped, as the reading starts at the end. We recommend to leave this option on false
+    # but lower the ignore_older value to release files faster.
+    #force_close_files: false
+
+  # Additional stdin prospector
+
+  # Configuration to use stdin input
+  #- input_type: stdin
+
+
+######################### Filebeat general configuration #############################
+
+filebeat:
```

Comment:
I quite like the idea of separating general config and prospectors. Should we even comment out "filebeat" here as all sub config options are commented out?